public/locales/en.json

```
@@ -99,7 +99,7 @@
   "poolSummaryStaked": "My Staked Value",
   "poolSummaryHaloEarned": "Unclaimed HALO Earned",
   "totalPoolValue": "Total Pool Value",
-  "tokenCardRewardDescription": "Every HALO token you earn is automatically staked in DESSERT to earn vesting rewards! Press Claim or go to the Vesting tab to manage rewards.",
+  "tokenCardRewardDescription": "Every HALO token you earn is automatically staked in Rainbow Pool to earn vesting rewards! Press Harvest or go to the Vesting tab to manage rewards.",
```

Shouldn't this be the name of the reward token?

```suggestion
  "tokenCardRewardDescription": "Every HALO token you earn is automatically staked in RAINBOW DROP to earn vesting rewards! Press Harvest or go to the Vesting tab to manage rewards.",
```

---

> Shouldn't this be the name of the reward token?

hmmm I think it should still say "staked in Rainbow Pool" because RAINBOW DROP is the token name.
Rainbow Pool is the product name for the Rainbow contract
Rainbow Drop is the token name
RNBW token symbol.

---

Yeah, but this is originally "staked in DESSERT", where DESSERT is the token name not the product name. So thought it would make more sense to replace it with the new token name as well - either RAINBOW DROP or Rainbow Drop

---

hmm I think Rainbow Pool makes more sense, even if it was dessert, I think Dessert Pool would have made more sense.. hehe

---

Well we could say automatically staked AS Rainbow Drop though, not IN.. that makes sense too..

---

@schystz i've changed it already

---

Jofo was the one who designed the original content so I think better to discuss with him what would be best?

---

Ok will discuss w Jofo, but yes, let's go with rainbow drop for now. I already changed it..